CONTRIBUTING.md

```diff
@@ -1,92 +1,46 @@
+**_IMPORTANT_**: We encourage contributions to the project from the community. We ask that you keep the following guidelines in mind when planning your contribution.
 
-  * Whether your contribution is for a bug fix or a feature request, **create an [Issue][create-issue]** and let us know
-   what you are thinking.
-  * **For bugs**, if you have already found a fix, feel free to submit a Pull Request referencing the Issue you created.
-  * **For feature requests**, we want to improve upon the library incrementally, which means small changes at a time. In
-   order ensure your PR can be reviewed in a timely manner, please keep PRs small, e.g. <10 files and <500 lines
-   changed. If you think this is unrealistic, then mention that within the issue and we can discuss it.
+* Whether your contribution is for a bug fix or a feature request, **create an [Issue](https://github.com/intelsdi-x/snap-plugin-collector-mesos/issues)** and let us know what you are thinking.
+* **For bugs**, if you have already found a fix, feel free to submit a Pull Request referencing the Issue you created. Include the `Fixes #` syntax to link it to the issue you're addressing.
+* **For feature requests**, we want to improve upon the library incrementally which means small changes at a time. In order to ensure your PR can be reviewed in a timely manner, please keep PRs small, e.g. <10 files and <500 lines changed. If you think this is unrealistic, then mention that within the issue and we can discuss it.
 
 Once you're ready to contribute code back to this repo, start with these steps:
 
-  * Fork the appropriate sub-projects that are affected by your change
-  * Clone the fork to `$GOPATH/src/github.com/intelsdi-x/`
+* Fork the appropriate sub-projects that are affected by your change.
+* Clone the fork to `$GOPATH/src/github.com/intelsdi-x/`:
 
     ```
-    $ git clone https://github.com/<yourGithubID>/<project>.git
+$ git clone https://github.com/<yourGithubID>/<project>.git
     ```
-
-  * Create a topic branch for your change and checkout that branch
+* Create a topic branch for your change and checkout that branch:
 
     ```
     $ git checkout -b some-topic-branch
     ```
+* Make your changes and run the test suite if one is provided.
+* Commit your changes and push them to your fork.
+* Open a pull request for the appropriate project.
+* Contributors will review your pull request, suggest changes, and merge it when it’s ready and/or offer feedback.
 
-  * Make your changes and run the test suite if one is provided (see below)
-  * Commit your changes and push them to your fork
-  * Open a pull request for the appropriate project
-  * Contributors will review your pull request, suggest changes, and merge it when it’s ready and/or offer feedback
-  * To report a bug or issue, please open a new issue against this repository
-
-If you have questions feel free to contact the [snap maintainers][snap-maintainers].
-
-### Vagrant development environment
```

Comment:
Seems to me this section would be useful for anyone attempting to develop the plugin in the included Vagrant environment, no?